src/Templates/src/templates/maui-mobile/App.xaml.cs

```diff
@@ -1,14 +1,28 @@
-﻿namespace MauiApp._1;
+﻿using Microsoft.Extensions.DependencyInjection;
+
+namespace MauiApp._1;
 
 public partial class App : Application
 {
+	public IServiceProvider ServiceProvider { get; private set; }
```

Comment:
I don't think this is correct, we don't want to create a new ServiceProvider here.

The announcement helper should get registered as a singleton into the ServiceProvider and then you add it to the ctor arguments on the VMs where it's used